db/schema.rb

```diff
@@ -28,6 +28,17 @@
     t.index ["code"], name: "index_local_authority_districts_on_code", unique: true
   end
 
+  create_table "que_jobs", primary_key: ["queue", "priority", "run_at", "job_id"], comment: "3", force: :cascade do |t|
```

Comment:
None of this schema looks too scary?

Reply:
Yeah I agree, I'll switch it to `sql` locally just to see what we'd miss by keeping as `ruby`

Reply:
If we stick with `ruby` be good to note this in the commit message for future us?

Reply:
This is what the change that is produced by Que looks like when switching to `sql`:

```
CREATE TABLE public.que_jobs (
    priority smallint DEFAULT 100 NOT NULL,
    run_at timestamp with time zone DEFAULT now() NOT NULL,
    job_id bigint NOT NULL,
    job_class text NOT NULL,
    args json DEFAULT '[]'::json NOT NULL,
    error_count integer DEFAULT 0 NOT NULL,
    last_error text,
    queue text DEFAULT ''::text NOT NULL
);

COMMENT ON TABLE public.que_jobs IS '3';

CREATE SEQUENCE public.que_jobs_job_id_seq
    START WITH 1
    INCREMENT BY 1
    NO MINVALUE
    NO MAXVALUE
    CACHE 1;

ALTER SEQUENCE public.que_jobs_job_id_seq OWNED BY public.que_jobs.job_id;
```
The only difference I see is the `CREATE SEQUENCE public.que_jobs_job_id_seq...` bit

Reply:
Yeah, looks like the `job_id` is always 0 with `:ruby` https://github.com/chanks/que/issues/148

Reply:
Not sure what to do about this, I'll start a thread on Slack

Reply:
Opted for [`delayed_job`](https://github.com/collectiveidea/delayed_job) instead